### PR TITLE
[docs] Add context to Nginx integration documentation

### PR DESCRIPTION
**Pull request by colleenmcginnis**
## What does this PR do?

From https://github.com/elastic/observability-docs/issues/1019:

>Let's audit the documentation for our top 5 integrations. The descriptions can be improved so users have a better understanding of what benefit they offer, how Elastic can help with the user's use case, and persuade them to install the integration.

This PR adds more context the Nginx integration including:
- Applies a new proposed structure to be applied across top integrations (overview, data types, requirements/prerequisites, setup, reference).
- Creates a soft division between guide-like content and reference-like content.
- Adds context to the "Overview":
    - Acknowledges the relationship between this integration and the larger ecosystem of Elastic products.
    - Provides a basic example to illustrate how this integration may be useful.
- Adds a "Data types" section:
    - Provides a high-level overview of the kind of data that is collected. (It's difficult to derive a summary from just the reference sections since they're so long.)
    - Reinforces what "Logs" and "Metrics" are in the context of the integration.
- Adds a "Requirements" section:
    - Reinforces how this integration fits into the larger Elastic system.
- Points the reader to the Getting started guide for step-by-step "Setup" instructions.

### For the reviewer

- The current doc says:
    >The Nginx `stubstatus` metrics was tested with Nginx 1.19.5 and are expected to work with all version >= 1.9.
    
    Should that be `>= 1.19` at the end? 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] Review by @bmorelli25 
- [ ] Review by TBD (Integrations team)

## Related issues

- https://github.com/elastic/observability-docs/issues/1019

### CODE REVIEW

File: packages/nginx/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.3"
+  changes:
+    - description: Update documentation with additional context for new users.
+      type: enhancement
+      link: TBD
```
Comment (colleenmcginnis) [inline packages/nginx/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/3393
```

**Comment by ollyhowell** [comment]
@akshay-saraswat could you take a look and review please?

**Comment by andresrc** [comment]
@elastic/obs-service-integrations can someone take a look at this one?

**Review by andresrc — APPROVED** [review]

**Review by akshay-saraswat — APPROVED** [review]
LGTM

File: packages/nginx/docs/README.md [hunk 2]
```diff
@@ -393,7 +429,11 @@ location /nginx_status {
 }
 ```
 
-It's highly recommended to replace `127.0.0.1` with your server’s IP address and make sure that this page accessible to only you.
+Replace `127.0.0.1` with your server’s IP address and make sure that this page accessible to only you.
+
+#### Tested versions
+
+The Nginx `stubstatus` stream was tested with Nginx 1.19.5 and is expected to work with all versions >= 1.9.
```
Comment (ishleenk17) [inline packages/nginx/docs/README.md:436]: versions > = 1.19

**Comment by colleenmcginnis** [comment]
Thanks @ishleenk17! I've addressed your comments. Can you please approve if it looks ok? (I cannot merge without an approval from a code owner.)

**Pull request merged**